packages/adapter-cloudflare/index.js

```diff
@@ -16,19 +16,19 @@ export default function (options = {}) {
 
 			const static_files = utils
 				.copy(config.kit.files.assets, target_dir)
-				.map((f) => f.replace(`${target_dir}/`, ''));
+				.map((f) => f.replace(`${target_dir}${sep}`, ''));
 
 			const client_files = utils
 				.copy(`${process.cwd()}/.svelte-kit/output/client`, target_dir)
-				.map((f) => f.replace(`${target_dir}/`, ''));
+				.map((f) => f.replace(`${target_dir}${sep}`, ''));
 
 			// returns nothing, very sad
 			// TODO(future) get/save output
 			await utils.prerender({
 				dest: `${target_dir}/`
 			});
 
-			const static_assets = [...static_files, ...client_files];
+			const static_assets = [...static_files, ...client_files].map((f) => f.replaceAll(sep, '/'));
```

Comment:
This requires Node 15, which is higher than SvelteKit's 14 minimum. Best to use a RegExp, `/[\\\/]+/g`